test/integration/targets/connection_sshchroot/test_connection.inventory

```diff
@@ -0,0 +1,7 @@
+[ssh]
+ssh-pipelining    ansible_ssh_pipelining=true
+[ssh:vars]
```

Comment:
Use this instead:

```ini
[sshchroot]
sshchroot-pipelining    ansible_ssh_pipelining=true
sshchroot-no-pipelining ansible_ssh_pipelining=false
[sshchroot:vars]
```

This changes the labels from `ssh` to `sshchroot` and includes both the pipelining and no pipelining tests.

Reply:
Renamed, but i think i'm not support `no-pipelining`,